server/sonar-server/src/main/java/org/sonar/server/source/index/FileSourcesUpdaterUtil.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.source.index;
+
+import com.google.common.base.Joiner;
+import org.elasticsearch.action.update.UpdateRequest;
+import org.sonar.server.db.DbClient;
+
+import javax.annotation.Nullable;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileSourcesUpdaterUtil {
+
+  private static final String SQL_ALL = "SELECT %s FROM file_sources WHERE data_type='%s' ";
+  private static final String AFTER_DATE_FILTER = " AND updated_at>?";
+  private static final String PROJECT_FILTER = " AND project_uuid=?";
+
+  private static final String[] FIELDS = {
+    "project_uuid",
+    "file_uuid",
+    "updated_at",
+    "binary_data"
+  };
+
+  private FileSourcesUpdaterUtil() {
+    // only static stuff
+  }
+
+  public static PreparedStatement preparedStatementToSelectFileSources(DbClient dbClient, Connection connection, String dataType, long afterDate, @Nullable String projectUuid) throws SQLException {
```

Comment:
This line is too long. Using the Eclipse Formatter, IDEA should have correctly format this class, it's strange that it didn't correctly formatted it.